learning/katas/python/Common Transforms/Filter/ParDo/task.py

```diff
@@ -18,18 +18,17 @@
 
 from log_elements import LogElements
 
-p = beam.Pipeline()
+with beam.Pipeline() as p:
 
 
-class FilterOutEvenNumber(beam.DoFn):
+  class FilterOutEvenNumber(beam.DoFn):
 
-    def process(self, element):
-        if element % 2 == 1:
-            yield element
+      def process(self, element):
+          if element % 2 == 1:
+              yield element
```

Comment:
This class could be created outside the scope of `with`. It is generic code that does not depend on a specific pipeline.

Reply:
Good call. Done.